src/pykeen/triples/triples_factory.py

```diff
@@ -709,26 +719,49 @@ def from_path_binary(
         """
         path = normalize_path(path)
         logger.info(f"Loading from {path.as_uri()}")
-        data = torch.load(path)
-        return cls(**data)
+        return cls(**cls._from_path_binary(path=path))
+
+    @classmethod
+    def _from_path_binary(
+        cls,
+        path: pathlib.Path,
+    ) -> MutableMapping[str, Any]:
+        # load base
+        data = dict(torch.load(path.joinpath("state.pth")))
+        # load numeric triples
+        data["mapped_triples"] = torch.as_tensor(
+            pd.read_csv(path.joinpath("numeric_triples.tsv.gz"), sep="\t").values,
+            dtype=torch.long,
+        )
+        return data
 
     def to_path_binary(
         self,
         path: Union[str, pathlib.Path, TextIO],
-    ) -> None:
+    ) -> pathlib.Path:
         """
         Save triples factory to path in (PyTorch's .pt) binary format.
 
         :param path:
             The path to store the triples factory to.
         """
         path = normalize_path(path)
-        torch.save(self._get_binary_state(), path)
+        path.mkdir(exist_ok=True, parents=True)
+
+        # store numeric triples
+        pd.DataFrame(
+            data=self.mapped_triples.numpy(),
+            columns=[LABEL_HEAD, LABEL_RELATION, LABEL_TAIL],
+        ).to_csv(path.joinpath("numeric_triples.tsv.gz"), sep="\t", index=False)
```

Comment:
lets make the strings for the filenames class variables

Reply:
0292ce6